assets/js/util/tracking/createDataLayerPush.js

```diff
@@ -13,11 +13,10 @@ import { DATA_LAYER } from './index.private';
 export default function createDataLayerPush( target ) {
 	/**
 	 * Pushes data onto the data layer.
-	 *
-	 * @param {...any} args Arguments to push onto the data layer.
+	 * Must use `arguments` internally.
```

Comment:
```suggestion
	 * Must use `arguments` internally. Using an ES6 spread operator
	   (`…args`) will cause tracking events to _silently_ fail. See:
	   https://github.com/google/site-kit-wp/issues/1181
```

This issue is super-obscure, so I feel needs/warrants all the extra explanation we can give 😄